docs/alerts/webhook-connections/jira-cloud.md

```diff
@@ -29,7 +29,7 @@ To send webhook alerts to Jira Cloud you need to include a Basic Authentication
 In Sumo Logic, Scheduled Searches and Monitors send alerts to other tools via webhook connections. To send alerts from Sumo Logic to Jira Cloud:
```

Comment:
```suggestion
In Sumo Logic, **Scheduled Searches** and **Monitors** send alerts to other tools via webhook connections. To send alerts from Sumo Logic to Jira Cloud:
```

Reply:
@JainM6 these aren't UI elements - don't need to be bold